### ISSUE TITLE
Calendar icon daily update

### ISSUE THREAD

**Issue by arainot** [pock/pock#36]
Would it be possible to have the calendar icon update with the actual current date?

**Comment by pock66** [comment]
yes please

**Comment by kopf** [comment]
This is unfortunately still a rather disorienting bug and imho shouldn't be closed.

**Comment by pigigaldi** [comment]
> This is unfortunately still a rather disorienting bug and imho shouldn't be closed.

I agree with you that this issue shouldn't be closed, but I won't call this a "bug", that's just how macOS treat the Calendar icon in certain contexts (see screenshot below for a clear example of what I'm talking about).
What we can do is create an ad-hoc icon for the Calendar app to solve this case 👍🏻 

<img width="944" alt="Screenshot 2022-01-29 at 17 07 33" src="https://user-images.githubusercontent.com/422119/151668342-4354373b-2ffc-48c6-a01b-7db018e4df33.png">

**Comment by kopf** [comment]
> What we can do is create an ad-hoc icon for the Calendar app to solve this case 

That would be fantastic.

**Comment by kopf** [comment]
Still not solved according to @pigigaldi 's comment, afaik
